Add Beatmap Spotlights Summer 2022 conclusion news post (#8289)
ppy/osu-wiki: news/2022-11-03-beatmap-spotlights-summer-2022-conclusion.md

---
layout: post
title: "Beatmap Spotlights Summer 2022 Conclusion & Players' Pick Voting"
date: 2022-11-03 11:00:00 +0000
---

The Summer 2022 season of the Beatmap Spotlights has come to an end⁠ — come check out the top 50 for each game mode, and vote on your favourite beatmaps from the season! Be sure to give your output on Beatmap Spotlights Survey as well.

![](/wiki/shared/news/banners/beatmap-spotlights.jpg)

Summer 2022 season is officially finished now! We faced some technical issues that had to be overcome but we're ready to deliver you the final rankings you all have been waiting for! This time around, we will also have you fill a quick survey to help us improve the Beatmap Spotlights workflow, so be sure to read on below!

## Survey

> It's been over two years since we launched the Spotlights in a league format. It's quite a handful of time to be around, and therefore we wanted hear from you - the playerbase on how do you feel about current state of Spotlights and whether you have some suggestions on how to improve the ongoing system!

Reviewer: ```suggestion
It's been over two years since we launched the Spotlights in a league format. It's quite a handful of time to be around, and therefore we wanted hear from you — the playerbase on how do you feel about current state of Spotlights and whether you have some suggestions on how to improve the ongoing system!
```

> It's been over two years since we launched the Spotlights in a league format. It's quite a handful of time to be around, and therefore we wanted hear from you — the playerbase on how do you feel about current state of Spotlights and whether you have some suggestions on how to improve the ongoing system!

Reviewer 2: ```suggestion
It's been over two years since we launched the Spotlights in a league format. It's quite a handful of time to be around, and therefore we wanted hear from you — the playerbase — on how you feel about current state of Spotlights and whether you have some suggestions on how to improve the ongoing system!
```

> You can fill it over [google forms](https://docs.google.com/forms/d/e/1FAIpQLSfUYNToOAnPDggnv98_u2pmDXZa-lKq8DU1QpA1aZRzw12azg/viewform), it's just a few questions, so it shouldn't take a lot of time.

Reviewer: ```suggestion
You can fill it over [Google forms](https://docs.google.com/forms/d/e/1FAIpQLSfUYNToOAnPDggnv98_u2pmDXZa-lKq8DU1QpA1aZRzw12azg/viewform), it's just a few questions, so it shouldn't take a lot of time.
```

> You can fill it over [Google forms](https://docs.google.com/forms/d/e/1FAIpQLSfUYNToOAnPDggnv98_u2pmDXZa-lKq8DU1QpA1aZRzw12azg/viewform), it's just a few questions, so it shouldn't take a lot of time.

Reviewer 2: ```suggestion
If you'd like to take part, just fill out [this quick Google form](https://docs.google.com/forms/d/e/1FAIpQLSfUYNToOAnPDggnv98_u2pmDXZa-lKq8DU1QpA1aZRzw12azg/viewform). It's only a few questions, so it shouldn't take a lot of time.
```
